prow/config/config.yaml

```diff
@@ -1103,11 +1103,22 @@ branch-protection:
                   - 'idc-jenkins-ci-tiflash/build'
                 strict: true
     chaos-mesh:
+      protect: true
```

Comment:
Please put the configuration at the branch level, we don't want the robot to hosted all branch configurations of all repos of the org.

Reply:
Got it. I will update now.

Reply:
Fixed in 46f0e7f.